lua-cjson: fix broken PKG_SOURCE_URL

bretello opened the pull request:
Current domain (kyne.com.au) has expired, so the current `PKG_SOURCE_URL` is non-existent.

See https://github.com/mpx/lua-cjson/commit/718f27293a981fb5e9e662e9aec0b7cf78317da6b

Signed-off-by: bretello <bretello@distruzione.org>

feckert commented:
@bretello please fix CI/CD issue https://github.com/openwrt/packages/actions/runs/9838982779/job/27164427254
by adding your first and last name to the 'Signed-off-by' line.
And also bump the PKG_RELEASE number https://github.com/openwrt/packages/blob/master/lang/lua-cjson/Makefile#L12

bretello commented:
@feckert done

feckert commented:
Link in the commit description does not work?
https://github.com/mpx/lua-cjson/commit/718f27293a981fb5e9e662e9aec0b7cf78317da6b

bretello commented:
Ops, it seems I added an extra `b` to the sha, I updated with the correct link (https://github.com/mpx/lua-cjson/commit/718f27293a981fb5e9e662e9aec0b7cf78317da6) thanks for bearing with me

feckert commented:
Merged thanks
